front/lib/api/assistant/participants.ts

```diff
@@ -109,6 +115,12 @@ export async function fetchConversationParticipants(
     fetchAllAgentsById(auth, [...agentConfigurationIds]),
   ]);
 
+  // if less agents than agentConfigurationIds, it means some agents are forbidden
```

Comment:
Here the check on length is used to determine whether some agents were forbidden. 

This is a compromise, since it's not the cleanest way to do this check. The alternative is refactoring all getAgentConfiguration(s) calls (~30 of them + potentially the parent callers) to return a Result type rather than directly the agent configuration. It's a cleaner option, but heavier--not only to do now but in code every time we'll call getAgentConfig(s), we'll have to unpack / handle which is not always useful.

Hesitating a bit there so happy to get the reviewers' opinions. Not opposed to go for the cleaner but heavier way.